packages/components/src/InputDate/InputDate.tsx

```diff
@@ -57,13 +62,14 @@ export function InputDate(inputProps: InputDateProps) {
         const { onChange, onClick, value } = activatorProps;
         const newActivatorProps = omit(activatorProps, ["activator"]);
 
-        const suffix = {
-          icon: "calendar",
-          ...(onClick && {
-            onClick: onClick,
-            ariaLabel: "Show calendar",
-          }),
-        } as Suffix;
+        const suffix =
+          inputProps.showIcon === undefined || inputProps.showIcon
```

Comment:
I think this can be re-written as `inputProps !=== false` and on the doc string for show icon add

`@default true`